packages/web-components/fast-components/src/number-field/number-field.definition.ts

```diff
@@ -11,106 +11,127 @@ export const fastNumberFieldDefinition: WebComponentDefinition = {
             attributes: [
                 {
                     name: "value",
-                    description: "The value attribute",
+                    title: "Value",
+                    description: "The HTML value attribute of the number field",
                     default: undefined,
                     required: false,
                     type: DataType.string,
                 },
                 {
                     name: "appearance",
-                    description: "The appearance attribute",
+                    title: "Appearance",
+                    description: "The number field's visual treatment",
                     default: "outline",
                     values: [{ name: "outline" }, { name: "filled" }],
                     type: DataType.string,
                     required: false,
                 },
                 {
                     name: "autofocus",
-                    description: "The autofocus attribute",
+                    title: "Autofocus",
+                    description:
+                        "Determines if the element should receive document focus on page load",
                     required: false,
                     type: DataType.boolean,
                     default: false,
                 },
                 {
                     name: "placeholder",
-                    description: "The placeholder attribute",
+                    title: "Placeholder",
+                    description:
+                        "Sets the placeholder value of the element, generally used to provide a hint to the user",
                     required: false,
                     type: DataType.string,
                     default: undefined,
                 },
                 {
                     name: "list",
-                    description: "The list attribute",
+                    title: "List",
```

Comment:
I think there was another `list` attribute I missed but I would suggest it also includes "ID"

```suggestion
                    title: "List ID",
```